tests_new/integration_tests/modules/s3_datasets/global_conftest.py

```diff
@@ -244,10 +317,7 @@ def temp_s3_dataset1(client1, group1, org1, session_env1, session_id, testdata):
 """
 
 
-def get_or_create_persistent_s3_dataset(
-    dataset_name, client, group, env, bucket=None, kms_alias=None, glue_database=None
-):
-    dataset_name = 'persistent_s3_dataset1'
+def get_or_create_persistent_s3_dataset(dataset_name, client, group, env, bucket=None, kms_alias='', glue_database=''):
```

Comment:
`dataset_name` is further used only for bucket naming and tags. Let's make it a dataset name as well. Otherwise all datasets has the same name `TestDatasetCreated`

Reply:
Done!